pandas/core/apply.py

```diff
@@ -1089,6 +1105,36 @@ def apply_series_numba(self):
         results = self.apply_with_numba()
         return results, self.result_index
 
+    def apply_series_bodo(self) -> DataFrame | Series:
+        if self.result_type is not None:
+            raise NotImplementedError(
+                "the 'bodo' engine does not support result_type yet."
+            )
+
+        if self.axis != 1 and not isinstance(self.func, str):
+            raise NotImplementedError(
+                "the 'bodo' engine only supports axis=1 for user-defined functions."
+            )
+
+        if self.args or self.kwargs:
+            raise NotImplementedError(
+                "the 'bodo' engine does not support passing additional args/kwargs "
+                "to apply function yet."
+            )
+
+        bodo = import_optional_dependency("bodo")
+
+        @bodo.jit(**self.engine_kwargs)
```

Comment:
Does `boto.jit` take the same keyword arguments as `numba.jit`? I couldn't easily find the API docs for `bodo.jit`